### ISSUE TITLE
Add coverage data to codecov

### ISSUE THREAD

**Issue by justinvp** [pulumi/pulumi-dotnet#162]
- [x] Unit test coverage of SDK (https://github.com/pulumi/pulumi-dotnet/pull/175)
- [ ] Language host coverage (blocked on https://github.com/pulumi/pulumi/issues/9780) -- This needs a large overhaul to make kill logic work, put on hold
